modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -129,28 +137,50 @@ else if (key.contains("objectRelationshipName")) {
 								relatedSchemaName);
 						}
 
-						_addObjectRelationshipPathItem(
-							key, objectRelationship, paths, relatedSchemaName);
+						if (Objects.equals(
+								objectRelationship.getType(),
+								ObjectRelationshipConstants.
+									TYPE_MANY_TO_MANY) ||
+							(Objects.equals(
+								objectRelationship.getType(),
+								ObjectRelationshipConstants.TYPE_ONE_TO_MANY) &&
+							 (objectRelationship.getObjectDefinitionId1() ==
+								 _objectDefinition.getObjectDefinitionId()))) {
+
+							_addObjectRelationshipPathItem(
+								key, objectRelationship, paths,
+								relatedSchemaName);
+						}
 					}
 
-					openAPI.getComponents(
-					).getSchemas(
-					).get(
-						_objectDefinition.getShortName()
-					).getProperties(
-					).put(
-						objectRelationship.getName(),
-						new Schema<Object>() {
-							{
-								setDescription(
-									StringBundler.concat(
-										"Information about the relationship ",
-										objectRelationship.getName(),
-										" can be embedded with ",
-										"\"nestedFields\"."));
-							}
-						}
-					);
+					if (GetterUtil.getBoolean(
+							PropsUtil.get("feature.flag.LPS-168462")) &&
```

Comment:
AFAIK, we could get the nestedEntities already without the necessity of any feature flag, right?

in that case, is this feature flag needed? :thinking: